include/net/nrf_cloud_rest.h

```diff
@@ -44,10 +44,12 @@ enum nrf_cloud_http_status {
 
 /** @brief nRF Cloud AGPS REST request types */
 enum nrf_cloud_rest_agps_req_type {
+	/** Request all assistance data */
 	NRF_CLOUD_REST_AGPS_REQ_ASSISTANCE,
+	/** Request only location (NRF_CLOUD_AGPS_LOCATION) */
```

Comment:
I think the nRF Cloud location services team is deprecating NRF_CLOUD_AGPS_LOCATION, since ground_fix covers that need now.